### ISSUE TITLE
[Feature] Command Line Arguments to Locate Data in a Specified Directory of the User's Choice

### ISSUE THREAD

**Issue by zahroc** [xmichelo/Beeftext#318]
It would be nice to be able to specify where the Data Folder resides.

Using the portable edition, the application bits may reside on the computer but the data resides on a flash-drive for security purposes.

i.e. 
BeefText.exe --datalocation "G:\_datafiles_\BeefText\"

**Comment by xmichelo** [comment]
Indeed. You can already move a few things from the preferences (combo list, backups), but not the entire Data folder in the portable Edition.
